App/src/pages/CreateGroup.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Container, FormControl, FormLabel, Heading, HStack, Input, Stack, Select } from "@chakra-ui/react";
+import logo from "../assets/logo.png";
+import { useForm } from "react-hook-form";
+import { useNavigate } from "react-router-dom";
+import { GroupCreateDTO, GroupResponseDTO } from "../services/group";
+import { CreateGroupCall } from "../services/group";
+import Button from "../components/Button";
+
+export default function CreateGroup() {
```

Comment:
I think it'd make sense for CreateGroup to have a back button to return to the manageable groups page

Reply:
I can add this

Reply:
Ok I added it